### PR TITLE
tests: fix regression in qa/tasks/ceph_master.py

### PR DESCRIPTION
**Pull request by smithfarm**
Fix regression introduced by https://github.com/ceph/ceph/pull/13194

### CODE REVIEW

File: qa/tasks/ceph_manager.py [hunk 1]
```diff
@@ -302,11 +302,12 @@ def kill_osd(self, osd=None, mark_down=False, mark_out=False):
             cmd = (prefix + "--op import --file {file}")
             cmd = cmd.format(id=imp_osd, file=exp_path)
             proc = imp_remote.run(args=cmd, wait=True, check_status=False)
-            output = proc.stderr.getvalue()
-            bogosity = "The OSD you are using is older than the exported PG"
-            if proc.exitstatus == 1 and bogosity in output:
-                self.log("OSD older than exported PG"
-                         "...ignored")
+            stderr_content = '' if (proc.stderr is None) else proc.stderr.getvalue()
```
Comment (zmc) [inline qa/tasks/ceph_manager.py]: `proc.stderr` is always going to be `None` here

File: qa/tasks/ceph_manager.py [hunk 2]
```diff
@@ -301,12 +301,14 @@ def kill_osd(self, osd=None, mark_down=False, mark_out=False):
...
             cmd = cmd.format(id=imp_osd, file=exp_path)
-            proc = imp_remote.run(args=cmd, wait=True, check_status=False)
-            output = proc.stderr.getvalue()
-            bogosity = "The OSD you are using is older than the exported PG"
-            if proc.exitstatus == 1 and bogosity in output:
-                self.log("OSD older than exported PG"
-                         "...ignored")
+            proc = imp_remote.run(args=cmd, wait=True, check_status=False,
+                                  stderr=StringIO())
+            stderr_content = proc.stderr.getvalue()
```
Comment (tchaikov) [inline qa/tasks/ceph_manager.py]: no need to define a `stderr_content`, it is not referenced otherwhere besides line 309.

File: qa/tasks/ceph_manager.py [hunk 3]
```diff
@@ -301,12 +301,14 @@ def kill_osd(self, osd=None, mark_down=False, mark_out=False):
...
+            if proc.exitstatus == 1:
+                bogosity = "The OSD you are using is older than the exported PG"
+                if bogosity in stderr_content:
```
Comment (tchaikov) [inline qa/tasks/ceph_manager.py]: `if bogosity in proc.stderr.getvalue()` is good enough.

**Comment by smithfarm** [comment]
`teuthology-suite -k distro --machine-type smithi --priority 101 --email ncutler@suse.com --ceph master --filter="$filter" --ceph-repo http://github.com/ceph/ceph.git --suite-repo http://github.com/smithfarm/ceph --suite-branch wip-fix-regression --suite rados`

**running** http://pulpito.front.sepia.ceph.com:80/smithfarm-2017-02-06_18:35:45-rados-master-distro-basic-smithi/

**Comment by smithfarm** [comment]
Addressed @tchaikov nit and scheduled another test:

**running** http://pulpito.front.sepia.ceph.com:80/smithfarm-2017-02-06_18:38:23-rados-master-distro-basic-smithi/

**Comment by yuriw** [comment]
and one more with -p 70

http://pulpito.front.sepia.ceph.com/yuriw-2017-02-06_20:42:26-rados-wip-yuri-testing_2017_2_6-distro-basic-smithi/

**Comment by smithfarm** [comment]
jenkins retest this please

**Comment by smithfarm** [comment]
Tests passed; now waiting for Jenkins.

**Pull request merged**